crates/forge/src/main.rs

```diff
@@ -36,6 +36,24 @@ struct Args {
     /// Stop test execution after first failed test
     #[arg(short = 'x', long)]
     exit_first: bool,
+
+    /// Number of fuzzer runs
+    #[arg(short = 'r', long, value_parser = validate_runs)]
+    fuzzer_runs: Option<u32>,
+
+    /// Seed for the fuzzer
+    #[arg(short = 's', long)]
+    fuzzer_seed: Option<u64>,
+}
+
+fn validate_runs(val: &str) -> Result<u32> {
```

Comment:
```suggestion
fn validate_runs(val: &str) -> Result<u32> {
```
The name suggests validation of finished runs

Reply:
It's in the main.rs which only handles entry point to the program. I don't think it would confuse anyone. It's used in the `Args` struct only and even accepts `str` as argument.

Reply:
I mean i mentioned it beacuse it confused me and i have to check the function body to see what it does. I didin't came up with some arbitrary "confusion possibility" I suggest the name`validate_fuzzer_runs_value`